### PR TITLE
docs: tweaks

### PR DESCRIPTION
**Pull request by matryer**
* Use US English 
* Add *Detected outliers* to the sorting page

### CODE REVIEW

File: docs/sources/ordering/_index.md [hunk 1]
```diff
@@ -24,16 +24,17 @@ You can use the **Asc/Desc** menu to change the direction of the sort.
...
-| Percentiles    | Sorts graphs by the nth percentile.                       |
+By default the graphs are sorted by **Most relevant** where we prioritize graphs with more volatile data. For example, the graphs with the most spikes or dips will be shown first.
+
+| Sort by option    | Description                                                 |
+| ----------------- | ----------------------------------------------------------- |
+| Most relevant     | Sorts graphs by the most volatile data.                     |
+| Detected outliers | Puts outliers first (those that look different to the rest) |
+| Widest spread     | Sorts graphs that contain the largest standard deviation.   |
+| Name              | Sorts graphs alphabetically by name.                        |
+| Highest spike     | Sorts graphs by the highest peaks.                          |
```
Comment (gtk-grafana) [inline docs/sources/ordering/_index.md:35]: There's also "Count" which sorts by total number of logs (i.e. Volume)

File: docs/sources/ordering/_index.md [hunk 2]
```diff
@@ -24,16 +24,17 @@ You can use the **Asc/Desc** menu to change the direction of the sort.
...
+| Lowest dip        | Sorts graphs by the lowest dips.                            |
```
Comment (JStickler) [inline docs/sources/ordering/_index.md:36]:
```suggestion
| Lowest dip        | Sorts graphs by the largest drop in a value.                            |
```

File: docs/sources/ordering/_index.md [hunk 3]
```diff
@@ -24,16 +24,17 @@ You can use the **Asc/Desc** menu to change the direction of the sort.
...
+| Percentiles       | Sorts graphs by the nth percentile.                         |
```
Comment (JStickler) [inline docs/sources/ordering/_index.md:37]:
```suggestion
| Percentiles       | Sorts graphs by the greatest percentile.                         |
```

**Comment by gtk-grafana** [comment]
Closing for inactivity

**Pull request closed without merging**